li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerImpl.java

```diff
@@ -150,6 +154,29 @@ public Object metricValue() {
       }
     };
 
+    MetricName offsetOutOfRangeCounterName = new MetricName(
+        "consumer-offset-out-of-range-counter",
+        "lnkd",
+        "counter of how many times the consumer reached OffsetOutOfRange or Invalid Offset exceptions.",
```

Comment:
nit: It should be `OffsetOutOfRangeException` or `NoOffsetForPartitionException`.